.vscode/settings.json

```diff
@@ -35,7 +35,7 @@
     "pre-commit-helper.runOnSave": "all hooks",
     "python.linting.mypyArgs": [],
     "mypy.targets": [
-        "src"
+        "app"
```

Comment:
I personally try to prevent to mix fixes for different bugs in one PR, have we agreed on a best practice?
The other option would be to leave the mypy setting with this PR the way it was, and create a new PR for it.

Reply:
I'm not aware of any best practice... I just additionally did this small fix, since it is so small...

I set it to the whole app-folder to let mypy also check the tests, because they also belong to the development IMHO